concepts/errors-handling/introduction.md

```diff
@@ -1,3 +1,70 @@
 # Introduction
 
-TODO: add introduction for errors-handling concept
+In Go, errors must be the last value returned in a multiple returns function.
+
+Always return the default value for the type, if returning an error:
```

Comment:
```suggestion
Always return the zero value for the type, if returning an error:
```
Possibly link to the [relevant sections](https://golang.org/ref/spec#The_zero_value) in the language specification as well if inline links are allowed in concept bodies.